docs/Language Feature Status.md

```diff
@@ -10,6 +10,7 @@ efforts behind them.
 
 | Feature | Branch | State | Developer | Reviewer | LDM Champ |
 | ------- | ------ | ----- | --------- | -------- | --------- |
+| [Interceptors](https://github.com/dotnet/csharplang/issues/7009) | [interceptors](https://github.com/dotnet/roslyn/tree/features/interceptors) | [In Progress](https://github.com/dotnet/roslyn/issues/67421) | [RikkiGibson](https://github.com/RikkiGibson) | [cston](https://github.com/cston) [jcouv](https://github.com/jcouv) | [RikkiGibson](https://github.com/RikkiGibson) |
```

Comment:
Please put it at the bottom of this table. Things that have been completed are put at the top.